.vale.ini

```diff
@@ -19,9 +19,3 @@ write-good.Passive = NO
 
 [formats]
 mdx = md
-
-[*.md]
-# These rules exist to allow JSX /HTML in markdown files.
-BlockIgnores = (?s) *(<[^>]*>)
-BlockIgnores = (?s) *(import.*?\n)
-BlockIgnores = (?s) *(export.*?\))
```

Comment:
I'm not sure what to do here to achieve the objective of the comment whilst letting it correctly not parse code blocks and such.

Reply:
I'm not entirely sure what you mean by this comment ?